app/modules/entities/components/layouts/authors_inline.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import { isNonEmptyArray } from '#app/lib/boolean_tests'
+  import { getEntitiesAttributesFromClaims } from '#entities/lib/entities'
+  import getBestLangValue from '#entities/lib/get_best_lang_value'
+  import Spinner from '#general/components/spinner.svelte'
+
+  export let entitiesUris = []
+
+  let authorsByUris
+  const firstAuthorsLimit = 5
+  const firstAuthorsUris = entitiesUris.slice(0, firstAuthorsLimit)
+
+  const waitingForAuthors = getAuthors()
+  async function getAuthors () {
+    if (isNonEmptyArray(entitiesUris)) {
+      const attributes = [ 'labels' ] as const
+      authorsByUris = await getEntitiesAttributesFromClaims(entitiesUris, attributes)
```

Comment:
This doesn't work
```suggestion
      ;({ entities: authorsByUris } = await getEntitiesAttributesByUris({ uris: entitiesUris, attributes }))
```

Reply:
Hum, it does to work locally for me, especially since `getEntitiesAttributesFromClaims` returns an already deconstructed `entities` array.